subprojects/docs/src/samples/java/multi-project-with-junit-5-tests/groovy/buildSrc/src/main/groovy/myproject.java-conventions.gradle

```diff
@@ -9,7 +9,7 @@ repositories {
     jcenter()
 }
 
-tasks.test.configure {
+tasks.named("test") {
```

Comment:
❓ At another place you did use `tasks.check.configure`. Why was it changed here to `tasks.named`?

Reply:
Turns out that `tasks.check.configure` is not lazy and we should always use `named(...)` in Groovy DSL. It's different in Kotlin DSL.